cartographer_rviz/cartographer_rviz/submaps_display.cc

```diff
@@ -66,6 +66,9 @@ SubmapsDisplay::SubmapsDisplay() : tf_listener_(tf_buffer_) {
       "All", true,
       "Whether submaps from all trajectories should be displayed or not.",
       trajectories_category_, SLOT(AllEnabledToggled()), this);
+  fade_out_start_distance_in_meters_ = new ::rviz::FloatProperty(
+      "Fade-out start distance in m", 1.,
```

Comment:
Conciseness: `Fade-out distance`? It's uncommon in RViz to mention that values are in SI units.
Also: `1.f`?